autopts/wid/bap.py

```
@@ -276,6 +277,16 @@ def hdl_wid_114(params: WIDParams):
         'BAP/BSRC/STR/BV-33-C': '48_6_1',
     }
 
+    stack = get_stack()
+
+    if stack.bap.wid_counter == 0:
+    	broadcast_id = stack.bap.broadcast_id
+    elif stack.bap.wid_counter == 1:
+    	broadcast_id = stack.bap.broadcast_id_2
+    else:
+    	# Invalid broadcast ID
+    	broadcast_id = 0x1000000
```

Suggest to raise an exception here. We want to fail loudly and not silently :)